Jenkinsfile

```diff
@@ -8,9 +8,23 @@ elifePipeline {
     node('containers-jenkins-plugin') {
         stage 'Build images', {
             checkout scm
+            sh "find build/critical-css -name '*.css' -type f -delete"
```

Comment:
as soon as there are multiple `sh` steps they are worth a separate script for decoupling from Jenkins and easy of debugging

Reply:
Now done to 1.

Reply:
Do we need this step? (ie are files left over from previous builds still present?)

Reply:
In general, they can be, failed builds do not clean the workspace to allow for debugging